README.md

```diff
@@ -43,6 +44,7 @@ Check.That(jsonElement).HasBoolProperty("boolProperty", expectedValue: true);
 Check.That(jsonElement).HasIntProperty("intProperty", 42);
 Check.That(jsonElement).HasLongProperty("longProperty", 2147483648);
 Check.That(jsonElement).HasNullProperty("nullProperty");
-Check.That(jsonElement).HasArrayProperty("arrayProperty", new[]{1, 2});
-Check.That(jsonElement).HasArrayPropertyEquivalentTo("arrayProperty", new[]{2,1});
+Check.That(jsonElement).HasGuidProperty("nullProperty", Guid.NewGuid());
```

Comment:
```suggestion
Check.That(jsonElement).HasGuidProperty("guidProperty", expectedGuid);
```
sorry I missed this one on the previous review (the `Guid.NewGuid()` is a bit more expressive but it kind of doesn't make sense for testing an existing value)